llpc/test/shaderdb/general/PipelineVsFs_TestPrimitiveID_First.pipe

```diff
@@ -0,0 +1,73 @@
+; Test primitiveID when provoking vertex mode is FIRST
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v --gfxip=10.1.0 %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} pipeline patching results
+; SHADERTEST: %{{[0-9]*}} = lshr i32 %{{[^,]*}}, 2
+; SHADERTEST: %{{[0-9]*}} = and i32 %{{[0-9]*}}, 2044
```

Comment:
Could you explain where these two constants (2 and 2044) come from? Looking at the vertex and fragment shaders, It's not obvious to me why these would be the expected values here and what could produce them.
Matching more instruction could help.

Same in the other test file.

Reply:
These two constants (2 and 2044)  are obtained after some instructions are optimized and are of little significance.
Origin instruction:   
  %6 = and i32 %esGsOffsets01, 511
  %7 = mul i32 %6, 4
  %8 = add i32 %7, 0
  %9 = getelementptr i8, i8 addrspace(3)* bitcast ([16384 x i32] addrspace(3)* @lds to i8 addrspace(3)*), i32 %8
  %10 = bitcast i8 addrspace(3)* %9 to i32 addrspace(3)*
  store i32 %gsPrimitiveId, i32 addrspace(3)* %10, align 4
  br label %.endWritePrimId